commit d3c96398eb988942b56dde8d70e3c2f2377eda22 (refs/changes/39/5139/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-11-27T10:05:40-08:00 (11 months ago)
    
    TRITON-986 CNAPI Server: header should include version
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/server.js b/lib/server.js
index e0cacef..ce6bd87 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -20,8 +20,11 @@ var request_seq_id = 0;
 
 
 function createServer(options) {
-    var cnapi = restify.createServer({
-        name: 'Compute Node API',
+    var cnapi;
+    var cnapiVersion = require(__dirname + '/../package.json').version;
+
+    cnapi = restify.createServer({
+        name: 'cnapi/' + cnapiVersion,
         log: options.log,
         handleUpgrades: true,
         handleUncaughtExceptions: false
diff --git a/package.json b/package.json
index b3196c5..69889d0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.16.0",
+  "version": "1.17.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
